plugins/thunderbolt/fu-thunderbolt-image.c

```diff
@@ -439,20 +493,36 @@ get_host_locations (guint16 id)
 }
 
 static const FuThunderboltFwLocation *
-get_device_locations (guint16 id)
+get_device_locations (guint16 id, const FuThunderboltFwObject *fw, GError **error)
 {
-	static const FuThunderboltFwLocation locations[] = {
+	static const FuThunderboltFwLocation AR[] = {
```

Comment:
lowercase pls.

Reply:
That's the "convention" used elsewhere in that file (AR, AR_LP, TR). No problem changing it to "ar" but then it looks different than the other places.

Reply:
Ahh, okay, no worries.